reference/interbit-ui-components/CONTRIBUTING.adoc

```diff
@@ -0,0 +1,187 @@
+= Contributing
```

Comment:
Please rename this file to `contributing.adoc`. We only use ALLCAPS filenames for GitBook-specific files.

Reply:
`SUMMARY.md`, `README.md` or `README.adoc` (following GitHub's convention if using these as index pages for each folder), and `GLOSSARY.md`. By naming the file `CONTRIBUTING.md`, the implication is that it contains instructions for contributing to the folder that contains it, whereas we're talking about contributions in an entirely separate repo.